tests/services/orders/validation_service/test_case_sample_rules.py

```diff
@@ -612,3 +617,42 @@ def test_existing_sample_from_outside_of_collaboration(
 
     # THEN the error should concern the added existing case
     assert isinstance(errors[0], SampleOutsideOfCollaborationError)
+
+
+def test_validate_sample_names_available(
+    valid_order: OrderWithCases,
+    store_with_multiple_cases_and_samples: Store,
```

Comment:
This test does not seem to need a store with multiple cases and samples. I suggest that you change this for a more simple (empty) store. Apparently, these big stores are an important factor in making our tests very slow (right? @mogmi ?)
The fixture `store` is empty and I think will work in this case
```suggestion
    store: Store,
```

Reply:
Smart!